src/openrct2/rct2/S6Importer.cpp

```diff
@@ -802,10 +803,7 @@ class S6Importer final : public IParkImporter
         dst.ProximityStart = { src.proximity_start_x, src.proximity_start_y, src.proximity_start_z };
         dst.CurrentRide = src.current_ride;
         dst.State = src.state;
-        dst.ProximityTrackType = src.proximity_track_type;
-        if (src.proximity_track_type == TrackElemType::RotationControlToggleAlias
-            && !RCT2TrackTypeIsBooster(_s6.rides[src.current_ride].type, src.proximity_track_type))
-            dst.ProximityTrackType = TrackElemType::RotationControlToggle;
+        dst.ProximityTrackType = RCT2TrackTypeToOpenRCT2(src.proximity_track_type, _s6.rides[src.current_ride].type);
```

Comment:
replays are crashing on this if the current ride is null then this will throw. Also `proximity_track_type` could be 0xFF to imply null this is a subtle bug from https://github.com/OpenRCT2/OpenRCT2/pull/13977.

Reply:
Hm, but 255 was also the quarter loop of the multi-dim. What if the proximity_track_type was that piece?

Reply:
Yeah it's a bit of a mess. I think though you can get away with just checking for null rides and just pretend that the track null issue doesn't exist.

Reply:
Hm, before I saw this I already pushed a commit that assumed that 255 was null (considering the special checks in RideRating, that seems safest to me). Is it okay to keep that?